tool/lib/globals.py

```diff
@@ -0,0 +1,24 @@
+import can
+from can.interfaces.pcan import PcanBus
+
+def initialize(): 
+    global bus 
```

Comment:
Why is interface handling broken out from `tool/lib/can_actions.py` to this new file in the first place? I fail to see how this improves the structure.

Assuming there is a good reason to do this, why is internal configuration exported as global definitions and why is it done as side effects of in an `initialize()` function?